src/chapters/05-more-about-components.md

```diff
@@ -68,11 +68,11 @@ Then, we will write a test to ensure all of the details are present. We will rep
 +    await render(hbs`<Rental />`);
 +
 +    assert.dom('article').hasClass('rental');
-+    assert.dom('article h3').containsText('Grand Old Mansion');
-+    assert.dom('article .detail.owner').containsText('Veruca Salt');
-+    assert.dom('article .detail.type').containsText('Standalone');
-+    assert.dom('article .detail.location').containsText('San Francisco');
-+    assert.dom('article .detail.bedrooms').containsText('15');
++    assert.dom('article h3').hasText('Grand Old Mansion');
++    assert.dom('article .detail.owner').hasText('Veruca Salt');
```

Comment:
I missed the fact that using `hasText` for lines 72-75 would fail because the selector includes the label/span text. Here, let's go with `includesText`?

Reply:
Isn't CI great 😄 I think I fixed!

Reply:
Haha, yep. I didn't know until a couple of days ago that you and Vaidehi had been writing code that generates the tutorial and output screenshots. Simply amazing.